blender/arm/props_ui.py

```diff
@@ -43,6 +44,94 @@
     arm.enable_reload(__name__)
 
 
+import bpy
+class ArmUIPropertyGroup(bpy.types.PropertyGroup):
+    show_editor_time_warnings: bpy.props.BoolProperty(
+           name="Show editor time warnings",
+           description="Template",
+           default=True)
+
+
+def label_multiline(text: str, context: bpy.types.Context, parent: bpy.types.UILayout, bullet: bool = False, bulletSymbol: str = "*"):
+    "Same as UILayout.label except with line splitting. If bullet is True we will put an asterix * before the first line and indent the rest."
+    text = str(text)
+    textListSplitByNewLines = text.split('\n')
+    import textwrap
+    for text in textListSplitByNewLines:
+
+        chars = int(context.region.width / 7)   # 7 pix on 1 character
+        amountOfSpaceNecessary = "  "
+        i = -1
+        while bulletSymbol.__len__() > i:
+            i += 1
+            amountOfSpaceNecessary += " "
+        if bullet:
+            wrapper = textwrap.TextWrapper(width=chars, initial_indent=f" {bulletSymbol} ", subsequent_indent=amountOfSpaceNecessary)
+        else:
+            wrapper = textwrap.TextWrapper(width=chars)
+        text_lines = wrapper.wrap(text=text)
+        for text_line in text_lines:
+            parent.label(text=text_line)
+
+
+from enum import Enum
+class WarningSeverityLevels(Enum): # Unfortunately this cant go inside EditorTimeWarnings as Python wont then let me reference it for default values in function parameters within EditorTimeWarnings.
+    ""
+    LOW = -1
+    NORMAL = 0
+    HIGH = 1
+    SEVERE = 2
+class EditorTimeWarnings():
+    "Do not instanciate me. Im responsible for keeping track of warnings that will later be displayed under the Armory Player panel."
+    class EditorTimeWarning():
+        "Instanciate me and append the instance to __listOfWarnings__. Im castable to string to get the formatted warning message."
+        def __init__(self, warningHeaderOrSettingsName: str, warningDescription: str, severityLevel: EditorTimeWarnings.SeverityLevels = WarningSeverityLevels.NORMAL) -> None:
+            self.warningHeaderOrSettingsName = warningHeaderOrSettingsName
```

Comment:
Armory uses snake_case for Python variable names (see [this](https://github.com/armory3d/armory/wiki/Code-Style#python)), maybe you can change this. It's pretty clean and understandable code nonetheless!

Reply:
Im not a big fan of snake case :P Youre welcome to rename my variables if you like

Reply:
Me neither, but that's how it is done in Armory and in a lot of Python projects. We should keep the project's code style as consistent as possible. Maybe VSCode has a little helper to change this automatically, otherwise I'll see if Pycharm (my Python IDE) has such a function and I can change it later.